server/api/controllers/withdrawal.js

```diff
@@ -139,6 +139,59 @@ const performWithdrawal = (req, res) => {
 		});
 };
 
+function performDirectWithdrawal(req, res) {
+	const { id: userId } = req.auth.sub;
+	const {
+		address,
+		currency,
+		amount,
+		network
+	} = req.swagger.params.data.value;
+
+	const domain = req.headers['x-real-origin'];
+	const ip = req.headers['x-real-ip'];
```

Comment:
We should log this IP for future audit purposes @LVMBDV . Check other similar cases.